### DISCUSSION TITLE
Proxmark3 RFID Reader for Phoniebox?

### DISCUSSION THREAD

**Discussion by Karli2727** [MiczFlor/RPi-Jukebox-RFID#1861] [Q&A]
Hello to everyone,

I was wondering if anyone already tried integrating a proxmark3 RFID Reader in Phoniebox? It seems to be a very promising piece of hardware because it can read 125khz as well as 13,56 mhz and is able to discover both ISO 14443 and ISO 15693 (Tonie Box Standard). 

I'm looking for a solution to keep my 125khz Tags and still be able to use exisiting Tonie Figurines (without their content, just as a trigger to do my Phoniebox Magic) and this seemed to be an affordable solution.  

I did some research but I couldn't find any post relating a proxmark3 reader to the phoniebox project, I just found some comments on a jooki project. So I thought I'll just order one and give it a try. 

I received my proxmark3 yesterday and plugged it in via USB and ran RegisterDevice.py, but it isn't discovered there. Does anyone have experience with the device? 

All the best

Karl

**Comment by apkuki** [comment]
that is a really neat pice of kit. i like it - my need to check where to get one an test that out, but need to finish https://github.com/MiczFlor/RPi-Jukebox-RFID/discussions/1860 first :-)

According to https://github.com/RfidResearchGroup/proxmark3/blob/master/doc/md/Installation_Instructions/Linux-Installation-Instructions.md you need to get through quite some steps and compile and flash the software and firmware for it. 
but once set up though it seems to act like a normal USB HID - so from there my educated guess is that it will come up as a normal USB Reader in Phoniebox.

Not tested though, just read through the article above.

**Reply by Karli2727** [reply]
Nice work with your Microbox, I'm looking forward to see how the final result will look like. 

I'm currently cycling through the instructions and am trying to get the Iceman Repo on my proxmark3 and will give feedback once I have it up and running. I initially thought it had some kind of usable standard firmware flashed out of the box but I guess I was too optimistic. 

If you end up ordering one yourself please make sure to get one with 512kb of internal memory, I'm currently in posession of one with only 256kb and am only a minute away from throwing it against a wall as I now have to disable features from the repo in order to get to a flash-rom-size that will actually fit on my device - of course without knowing what half of the features do that I'm currently disabling and if I'm going to need them in the future. 

But why should it be easy, right?
